dll/directx/wine/d3dcompiler_43/CMakeLists.txt

```diff
@@ -15,19 +15,21 @@ list(APPEND SOURCE
     main.c
     preproc.c
     reflection.c
-    utils.c
-    precomp.h
-    ${CMAKE_CURRENT_BINARY_DIR}/d3dcompiler_43_stubs.c)
+    utils.c)
 
-add_library(d3dcompiler_43 MODULE
-    ${SOURCE}
+list(APPEND PCH_SKIP_SOURCE
     asmshader.tab.c
     asmshader.yy.c
     hlsl.tab.c
     hlsl.yy.c
     ppy.tab.c
-    ppl.yy.c
+    ppl.yy.c)
+
+add_library(d3dcompiler_43 MODULE
+    ${SOURCE}
+    ${PCH_SKIP_SOURCE}
     version.rc
+    ${CMAKE_CURRENT_BINARY_DIR}/d3dcompiler_43_stubs.c
```

Comment:
In dhcpcsvc you have the stubs file in PCH_SKIP_SOURCE. I suppose technically that's better?

Reply:
Basically the idea back then was that we'd make use of windef.h (from stubs.h) being part of the PCH almost always, except when we get compile issues from including the stub files. What do you think?

Reply:
Ah. Yep that makes sense!